docs/docs-ref-autogen/excel/excelscript/excelscript.clearapplyto.yml

```
@@ -4,7 +4,29 @@ uid: 'ExcelScript!ExcelScript.ClearApplyTo:enum'
 package: ExcelScript!
 fullName: ExcelScript.ClearApplyTo
 summary: ''
-remarks: ''
+remarks: |-
+
+
+  #### Examples
+
+  ```TypeScript
+  /**
+   * This script removes an extra formatting that's been applied to a table. 
+   * This leaves only the base table style effects.
+   * Any formatting outside of the table will be left as is.
+   */
+  function main(workbook: ExcelScript.Workbook) {
+    // Get the first table on the current worksheet.
+    let worksheet = workbook.getActiveWorksheet();
+    let table = worksheet.getTables()[0];
+
```

```suggestion
if (table == undefined) return;
```
Check that a table existed to avoid error when no tables are on the worksheet.

---

@davidchesnut We've had this debate on the Office Scripts team about error checking in samples. I can be persuaded back, but we landed on avoiding extra error checking. The rationale was that it makes the samples larger and draws focus from what the sample is trying to illustrate. The macro recorder also doesn't add error checking, so scripts based on that won't look like our samples.

I am going to leave this as is for now, but let's plan on discussing this at a future team meeting.

---

Ok. thanks!